test/WebJobs.Script.Tests.Analyzers/WebJobsAttributeAnalyzerTests.cs

```diff
@@ -0,0 +1,88 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Xunit;
+using AnalyzerTest = Microsoft.CodeAnalysis.CSharp.Testing.CSharpAnalyzerTest<Microsoft.Azure.Functions.Analyzers.WebJobsAttributeAnalyzer, Microsoft.CodeAnalysis.Testing.Verifiers.XUnitVerifier>;
+using Verify = Microsoft.CodeAnalysis.CSharp.Testing.XUnit.AnalyzerVerifier<Microsoft.Azure.Functions.Analyzers.WebJobsAttributeAnalyzer>;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Testing;
+using System.Collections.Immutable;
+
+namespace WebJobs.Script.Tests.Analyzers
+{
+    public class WebJobsAttributeAnalyzerTests
+    {
+        [Fact]
+        public async Task ValidFunctionName_NoDiagnostic()
+        {
+            string testCode = @"
```

Comment:
nit: Could this be extracted to a separate method like GetTestCode(string functionName), to make the test more readable?

Reply:
Yes; this approach worked when there were only a few tests, but now it's getting unwieldy. There are some other possible approaches, such as loading the code from .cs files in the test project. I'd like to flag this as a near-future enhancement (before things get too out of hand) and make those changes in a separate PR.